3/ollie-charles-captions.srt

```diff
@@ -4773,11 +4777,11 @@ To me, the biggest source of expense
 
 1095
 01:10:15,250 --> 01:10:16,902
-feels like this neighbors function.
+feels like this neighbours function.
```

Comment:
Yes, I've tried to only change the neighbors to neighbours when he discusses the function explicitly. They als talk a lot about the concept of neighbors and I've tried to leave those in American spelling